### PR TITLE
[PDI-11537] Export Linked Resources: PMR Mapper and Reducer transformations not exported

### PR DESCRIPTION
**Pull request by stepanovdg**
@dkincade Details in the letter. Aliaksei Simkin is cheking it now.

### CODE REVIEW

**Comment by AliakseiSimkin** [comment]
This is not working on DI Server if user launch PMR job using "Execute Remotely", check "Pass export to  remote server" and select DI server or Carte server. (6.0.0.0-248 build was used)

**Comment by stepanovdg** [comment]
@dkincade Seems this one should be merged.

**Comment by dkincade** [comment]
We are going to focus on product stability before we can process any pull requests ... we will come back to this in a few hours

**Comment by dkincade** [comment]
@mattyb149 @brosander for your review ...

**Comment by mattyb149** [comment]
looks ok to me but I didn't run unit tests or checkstyle, will let @brosander have a crack at it

File: legacy/src/main/java/org/pentaho/di/job/entries/hadooptransjobexecutor/JobEntryHadoopTransJobExecutor.java [hunk 1]
```diff
@@ -754,13 +756,10 @@ public Result execute( Result result, int arg1 ) throws KettleException {
       String numMapTasksS = environmentSubstitute( numMapTasks );
       try {
         if ( Integer.parseInt( numMapTasksS ) < 0 ) {
-          throw new KettleException(
-              BaseMessages.getString( PKG, "JobEntryHadoopTransJobExecutor.NumMapTasks.Error" ) );
-        }
-      } catch ( NumberFormatException e ) {
-        if ( log.isDebug() ) {
```
Comment (brosander) [inline legacy/src/main/java/org/pentaho/di/job/entries/hadooptransjobexecutor/JobEntryHadoopTransJobExecutor.java]: @stepanovdg why are we removing the number format logging when loglevel is debug?

**Comment by stepanovdg** [comment]
Sorry, but I cannot remember the reason why it was made.  Has rebased some old branch. So it may be some merge issue. Definetely I have no ability to resend pull-request now. Soo could you please correct this, or I will send it tommorow if it can wait.

**Comment by dkincade** [comment]
I am sure it can wait for tomorrow...

**Comment by stepanovdg** [comment]
Checked resent: https://github.com/pentaho/big-data-plugin/pull/531 and https://github.com/pentaho/big-data-plugin/pull/530

**Pull request closed without merging**